src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -175,33 +173,12 @@ private boolean areQuestionNumbersConsistent(List<FeedbackQuestionAttributes> qu
                 copiableQuestions.addAll(questions);
             }
         }
-        Collections.sort(copiableQuestions, new Comparator<FeedbackQuestionAttributes>() {
-            @Override
-            public int compare(FeedbackQuestionAttributes q1, FeedbackQuestionAttributes q2) {
-                int order = q1.courseId.compareTo(q2.courseId);
-                if (order != 0) {
-                    return order;
-                }
-
-                order = q1.feedbackSessionName.compareTo(q2.feedbackSessionName);
-                if (order != 0) {
-                    return order;
-                }
-
-                FeedbackQuestionDetails q1Details = q1.getQuestionDetails();
-                FeedbackQuestionDetails q2Details = q2.getQuestionDetails();
 
-                String q1DisplayName = q1Details.getQuestionTypeDisplayName();
-                String q2DisplayName = q2Details.getQuestionTypeDisplayName();
-
-                order = q1DisplayName.compareTo(q2DisplayName);
-                if (order != 0) {
-                    return order;
-                }
-
-                return q1Details.getQuestionText().compareTo(q2Details.getQuestionText());
-            }
-        });
+        copiableQuestions.sort(Comparator.comparing((FeedbackQuestionAttributes copiableQuestion) ->
+                copiableQuestion.courseId)
+                .thenComparing(copiableQuestion -> copiableQuestion.feedbackSessionName)
+                .thenComparing(copiableQuestion -> copiableQuestion.getQuestionDetails().getQuestionTypeDisplayName())
+                .thenComparing(copiableQuestion -> copiableQuestion.getQuestionDetails().getQuestionText()));
```

Comment:
`copiableQuestion` -> `question`? At this scope `copiable` does not seem to value-add.